Octokit/Clients/Enterprise/EnterpriseAdminStatsClient.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+
+namespace Octokit
+{
+    /// <summary>
+    /// A client for GitHub's Enterprise Admin Stats API
+    /// </summary>
+    /// <remarks>
+    /// See the <a href="https://developer.github.com/v3/enterprise/admin_stats/">Enterprise Admin Stats API documentation</a> for more information.
+    ///</remarks>
+    public class EnterpriseAdminStatsClient : ApiClient, IEnterpriseAdminStatsClient
+    {
+        public EnterpriseAdminStatsClient(IApiConnection apiConnection)
+            : base(apiConnection)
+        { }
+
+        /// <summary>
+        /// Gets GitHub Enterprise statistics (must be Site Admin user).
+        /// </summary>
+        /// <remarks>
+        ///https://developer.github.com/v3/enterprise/admin_stats/#get-statistics
+        /// </remarks>
+        /// <returns>The <see cref="AdminStats"/> collection for the requested <see cref="AdminStatsType"/> type.</returns>
+        [SuppressMessage("Microsoft.Globalization", "CA1308:NormalizeStringsToUppercase")]
+        public async Task<AdminStats> GetStatistics(AdminStatsType type)
```

Comment:
@Haacked thoughts on how we could simplify this method?

For reference, [here's](https://developer.github.com/v3/enterprise/admin_stats/#get-statistics) the signature from the docs:

<img width="663" src="https://cloud.githubusercontent.com/assets/359239/12380791/5af70c00-bdcf-11e5-996b-dc098512e1ca.png">

And it seems like we're stuck with getting a loose dictionary of values irrespective of the endpoint we use.